sdk/digitaltwins/Azure.DigitalTwins.Core/samples/Readme.md

```diff
@@ -297,16 +296,38 @@ AsyncPageable<BasicDigitalTwin> asyncPageableResponseQueryBuilder = client.Query
 
 ### Build ADT Queries
 
-Build an [Azure DigitalTwins Query Store Language](https://docs.microsoft.com/azure/digital-twins/concepts-query-language) using an `DigitalTwinsQueryBuilder`. When using a `Where` clause, conditions are separated from the `Where` keyword (see `whereIsOfModel`)
+Build an [Azure DigitalTwins Query](https://docs.microsoft.com/azure/digital-twins/concepts-query-language) using a `DigitalTwinsQueryBuilder`. Two different objects entitled `DigitalTwinsQueryBuilder` exist in two namespaces, each with their own unique functionality and features. We encourage testing both objects and determining which works best given a specific set of needs.
```

Comment:
We need a note (if you have it somewhere, please point me to it) that says the sub-namespaces are being used, only in this preview, to distinguish between the two implementations. If/when we officially support a query builder, it will live in the base namespace.

Reply:
I think they are mentioned in specific descriptions of each approach. Nick can make it more clear.

Reply:
`It should be noted that though both versions of the DigitalTwin query builder currently (August 2021) reside in their own respective namespaces, if/when a DigitalTwins query builder is officially supported it will be a single object that in one namespace.`